src/pages/basic-management/depart/list/Index.js

```diff
@@ -0,0 +1,206 @@
+import React, { PureComponent, Fragment } from 'react';
+import router from 'umi/router';
+import { connect } from 'dva';
+import {
+    List,
+    Card,
+    Row,
```

Comment:
Codacy found an issue: ['Row' is defined but never used.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)